doc/rel_notes/0.9.0.txt

```diff
@@ -59,6 +59,9 @@ Tools
    when only a subset of cameras are computed and we do not want a mix of
    old and new camera files.
 
+ * Added command line arguments to get help (-h, --help) and to open directly a
+   config file at launch (-i, --input [PATH])
```

Comment:
The notes in this file are grouped by which part of the code they address.  Could you please move this item up under the section titled "Visualization Application".  That section corresponds to the GUI code, while "Tools" covers changes to the command line tools.  Thanks!